gammapy/modeling/parameter.py

```diff
@@ -357,26 +357,27 @@ def _step(self):
     # TODO: possibly allow to set this independently
     @property
     def conf_min(self):
-        """Confidence minimum value as a `float`.
-
-        Return parameter minimum if defined, otherwise return the scan_min.
-        """
+        """Confidence minimum value as a `float`."""
         if not np.isnan(self.min):
             return self.min
         else:
-            return self.value - self._step * self.scan_n_sigma
+            min_ = self.value - self._step * self.scan_n_sigma
+            large_step = np.maximum(self._step, np.abs(self.value))
+            min_ = np.minimum(min_, -large_step * 1e5)
+            return min_
 
     # TODO: possibly allow to set this independently
     @property
     def conf_max(self):
-        """Confidence maximum value as a `float`.
+        """Confidence maximum value as a `float`."""
```

Comment:
```suggestion
        """Confidence maximum value as a `float`.
        Return parameter maximum if defined, otherwise  a default is estimated from value and error.
        """
```